scripts.js

```diff
@@ -40,89 +33,89 @@ var hexCodeThree = document.querySelector('.hex-code-3');
 var hexCodeFour = document.querySelector('.hex-code-4');
 var hexCodeFive = document.querySelector('.hex-code-5');
 
+// var palette = document.querySelector('.palette');
+// var colorProps = document.querySelectorAll('.color-props');
 var savedPaletteSection = document.querySelector('.saved-section');
-var colorProps = document.querySelectorAll('.color-props');
 
-window.addEventListener('load', createPalette);
-// window.addEventListener('load', generateColors);
+window.addEventListener('load', makeNewPalette);
 
-newPaletteButton.addEventListener('click', generateColors);
+newPaletteButton.addEventListener('click', makeNewPalette);
 savePaletteButton.addEventListener('click', savePalette);
-// lockButton.addEventListener('click', lockColor);
+
+savedPaletteSection.addEventListener('click', function(event) {
+   if (event.target.className === 'trash-icon') {
+     for (var i = 0; i < savedPalettes.length; i++) {
+       if (`${savedPalettes[i].id}` === event.target.id) {
+         savedPalettes.splice(i, 1);
+       }
+     }
+   }
+  displaySavedPalettes();
+ });
 
 colorOneDiv.addEventListener('click', function(event) {
   if (event.target.className === 'lock-color-1' || 'unlock-color-1') {
-    lockButtonOne.classList.toggle('hidden')
-    unlockButtonOne.classList.toggle('hidden')
+    lockButtonOne.classList.toggle('hidden');
+    unlockButtonOne.classList.toggle('hidden');
   }
```

Comment:
I believe you don't actually need the comparison here on line 57. There's no scenario where you would want to click the lock and have it not toggle its visibility. That's the beauty of `.toggle` over `.classList.add or classList.remove`.